test/e2e/gke_local_ssd.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apimachinery/registered"
+	"k8s.io/kubernetes/pkg/util"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = framework.KubeDescribe("GKE local SSD [Feature:GKELocalSSD]", func() {
+
+	f := framework.NewDefaultFramework("localssd")
+
+	It("should write and read from node local SSD [Feature:GKELocalSSD]", func() {
+		framework.Logf("Start local SSD test")
+		createNodePoolWithLocalSsds("np-ssd")
+		doTestWriteAndReadToLocalSsd(f)
+	})
+})
+
+func createNodePoolWithLocalSsds(nodePoolName string) {
+	framework.Logf("Create node pool: %s with local SSDs in cluster: %s ",
+		nodePoolName, framework.TestContext.CloudConfig.Cluster)
+	out, err := exec.Command("gcloud", "alpha", "container", "node-pools", "create",
```

Comment:
Can you file an issue against me to change this to the GA track? gcloud changes should be going out soon.